src/main/java/UITestFramework/CreateSession.java

```diff
@@ -44,7 +47,30 @@
 	Properties configProp = new Properties();
 	String OS;
 
+	public WebDriver getDriver() {
+        return driver;
+}
+
+@BeforeMethod
+public void createDriver() {
+        WebDriver driver= getDriver();
+}
 
+@AfterMethod
```

Comment:
@shivam198smart @AfterMethod already present. Please add to it if you want anything new else I think we should remove it too.

Reply:
Removed. It wasn't necessary.